src/scripts/ui/article-vote.js

```diff
@@ -3,12 +3,14 @@ window.addEventListener('DOMContentLoaded', () => {
   const $articleVoteUp = document.querySelector('.article-vote-up');
   const $articleVoteDown = document.querySelector('.article-vote-down');
 
-  const hideVoteBtns = () => {
-    $articleVoteQuestion.innerText = 'Thanks for your feedback!';
-    $articleVoteUp.style.display = 'none';
-    $articleVoteDown.style.display = 'none';
-  };
+  if ($articleVoteQuestion) {
```

Comment:
Was this meant for a separate story?

Reply:
There isn't a story for it. It is just general cleanup because it was causing an error in the console, which was making it a bit hard to debug issues. The error will appear on any page that isn't an article.